sabnzbd/cfg.py

```diff
@@ -278,6 +278,7 @@ def validate_server(value):
 x_frame_options = OptionBool("misc", "x_frame_options", True)
 require_modern_tls = OptionBool("misc", "require_modern_tls", False)
 num_decoders = OptionNumber("misc", "num_decoders", 3)
+improve_final_filenames = OptionBool("misc", "deobfuscate_final_result", False)
```

Comment:
For consistency, the variable name and the `name` should be the same. Otherwise it's very confusing to find stuff in the code when looking at options in the `sabnzbd.ini`.

Reply:
done